### DISCUSSION TITLE
Related Field Shows No Data - Urgent Please Help

### DISCUSSION THREAD

**Discussion by simple-AMS** [YetiForceCompany/YetiForceCRM#14014] [How to] [answered]
Hello @mariuszkrzaczkowski or anyone that can help me with this I would greatly appreciate it. As you can see from the screenshots below, when I click on the magnifying glass to display the data from the Agent list (Employees module) nothing shows up at all. 

![1](https://user-images.githubusercontent.com/42642523/103878659-58cecc00-509c-11eb-95c0-8c0089ac79ff.png)

![2](https://user-images.githubusercontent.com/42642523/103878664-5bc9bc80-509c-11eb-9341-b7aa00378376.png)

Clearly you can see there's over 11 thousand records so the info is there but it won't let you select an active agent record. This worked perfectly in last version 5.3.0 and all I have done is imported the Agent list data into the new stable install 6.1.0 

The server has no issues whatsoever and ran the previous version flawlessly. No errors in the System Warnings area or anything. How can I get this working again? Thanks so much everyone :) 

![3](https://user-images.githubusercontent.com/42642523/103878673-5e2c1680-509c-11eb-92be-77db157fbba9.png)

**Comment by rskrzypczak** [comment, marked as answer]
Check this: https://github.com/YetiForceCompany/YetiForceCRM/issues/13799

**Reply by simple-AMS** [reply]
@rskrzypczak Just what the doctor ordered mate. Thanks so very much, that was it. I forgot I had added a Picklist Status for Agents (Employees) that said Active. I went into the language management, changed the Field Name back to Employees which re-added the Picklist status Employee.  Then I deleted the old Active picklist and had everything point back to Employees and from the Languages renamed the now Employee status to Active and voila, all Active Agent records show up as they should.
